src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/domain/Oppfolgingstilfelle.kt

```diff
@@ -23,7 +23,7 @@ fun Oppfolgingstilfelle.passererAktivitetskravVurderingStoppunkt(): Boolean =
     durationInWeeks() >= AKTIVITETSKRAV_VURDERING_STOPPUNKT_WEEKS
 
 fun Oppfolgingstilfelle.toAktivitetskravVurdering(): AktivitetskravVurdering {
-    return if (gradertAtTilfelleEnd == true) {
+    return if (this.isGradertAtTilfelleEnd()) {
```

Comment:
Jeg tror du kan droppe `this`? 🤔 Med mindre du synes det er lettere å lese sånn det er nå da 👍